ReactWindows/ReactNative/Views/TextInput/ReactPasswordBoxManager.cs

```diff
@@ -0,0 +1,109 @@
+﻿using ReactNative.Reflection;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using ReactNative.Views.Text;
+using System;
+using Windows.UI.Text;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace ReactNative.Views.TextInput
+{
+    class ReactPasswordBoxManager : BaseViewManager<PasswordBox, ReactTextInputShadowNode>
+    {
+        public override string Name
+        {
+            get
+            {
+                return "PasswordBox";
+            }
+        }
+
+        #region CSS
+
+        [ReactProp("passwordRevealMode")]
+        public void SetPasswordRevealMode(PasswordBox view, string revealModeString)
+        {
+            var revealMode = EnumHelpers.ParseNullable<PasswordRevealMode>(revealModeString);
+            view.PasswordRevealMode = revealMode ?? PasswordRevealMode.Peek;
+        }
+
+        /// <summary>
+        /// Sets the font size on the <see cref="PasswordBox"/>.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="fontSize">The font size.</param>
+        [ReactProp(ViewProps.FontSize)]
+        public void SetFontSize(PasswordBox view, double fontSize)
```

Comment:
Weird, if this is how the ReactTextInputManager is working, then it may actually not be working.  I thought you could only have a prop on either the view manager or the shadow node...